### DISCUSSION TITLE
PHP 8 curly braces is no longer supported

### DISCUSSION THREAD

**Discussion by bartollajos** [mpdf/mpdf#1796] [General]
I need help regarding PHP 8. Got this error.

Fatal error: Array and string offset access syntax with curly braces is no longer supported in ...Mpdf.php on line 2187

**Comment by finwe** [comment]
Do you have the last version of mPDF? This should not happen. Otherwise, share an example code and details of your setup.

**Reply by bartollajos** [reply]
![error](https://user-images.githubusercontent.com/32515803/212647162-42eda5bc-0236-42ee-863a-4100d3c473d7.jpg)
I updated mpdf src folder. Now. I get this error.

**Reply by finwe** [reply]
updated how? this would not happen when using composer correctly.
